app/src/main/java/net/squanchy/tweets/TweetsPageView.java

```diff
@@ -69,32 +73,40 @@ protected void onAttachedToWindow() {
         initList();
     }
 
+    @Override
+    protected void onDetachedFromWindow() {
+        super.onDetachedFromWindow();
+        subscription.dispose();
+    }
+
     private void initList() {
```

Comment:
@fourlastor says "as long as you remember to call the methods because they don't get magically called". I suppose it means see how the other `LifecycleViews` are handled.